jscomp/runtime/js.ml

```diff
@@ -94,6 +94,9 @@ external log3 : 'a -> 'b -> 'c -> unit = "log"
 external log4 : 'a -> 'b -> 'c -> 'd -> unit = "log" 
 [@@bs.val] [@@bs.scope "console"]
 (** A convenience function to log *)
+external logN : 'a array -> unit = "log"
```

Comment:
would also help with confusion around whether this is a method like logn (usually differentiated by belonging to console or math)